src/gc.cr

```diff
@@ -66,7 +66,9 @@ module GC
     gc_no : UInt64,
     markers_m1 : UInt64,
     bytes_reclaimed_since_gc : UInt64,
-    reclaimed_bytes_before_gc : UInt64
+    reclaimed_bytes_before_gc : UInt64,
+    expl_freed_bytes_since_gc : UInt64,
```

Comment:
I know this is is the same naming as Boehm but should this be more clear

maybe something like `explicitly_freed_bytes_since_gc`

Reply:
We're going with the library names everywhere else, so it's fine to keep that here as well.